libs/utils/callCENNZ.ts

```diff
@@ -0,0 +1,52 @@
+import { CENNZNET_SIGNER } from "@/libs/constants";
+import { Api, SubmittableResult } from "@cennznet/api";
+import Keyring from "@polkadot/keyring";
+import { BigNumberish, BytesLike, utils } from "ethers";
+
+export const callCENNZ = async (
+	api: Api,
+	requestId: number,
+	returnData: BytesLike,
+	blockNumber: BigNumberish,
+	blockTimestamp: number
+): Promise<SubmittableResult> => {
+	const returnDataLength = utils.hexDataLength(returnData);
+
+	if (returnDataLength < 32) throw { code: "INVALID_RETURN_DATA" };
+
+	const signer = new Keyring({ type: "sr25519" }).addFromSeed(
```

Comment:
we might not want to create the wallet for every job... can pull this one out as CENNZNET_SIGNER is constant..